src/models/database.py

```diff
@@ -0,0 +1,98 @@
+import asyncio
+import logging
+from functools import wraps
+from pathlib import Path
+from typing import List
+
+import aiosqlite
+
+
+"""/////////////////UTILS///////////////////////"""
+
+
+def get_db_folder():
+    db_dir = Path.cwd() / "db"
+    db_dir.mkdir(exist_ok=True)  # create if folder does not exist
+    return db_dir
+
+
+"""/////////////////////////////////////////////"""
+
+
+class Database:
+    """A wrapper for a SQLite Database"""
+
+    def __init__(
+        self, commands: List[str], db_name: str, db_folder: Path = get_db_folder()
+    ):
+        """Initialise the SQLite Database. must include .sqlite file extension in Database name"""
+        path = db_folder / db_name
+        path.touch(exist_ok=True)  # create if Database does not exist
+        self.db_path = path.resolve()
+        self.name = db_name
+        asyncio.run(
+            self.initialise_database(commands)
+        )  # Catastrophic error: will crash if initialise_Database fails
+
+    def crash_handler(func):
```

Comment:
Should have self as the first argument.
```suggestion
    def crash_handler(self, func):
```

Reply:
Function will still have the same decorator signature `@Database.crash_handler`

Reply:
Self is introducing new problems into the code. As it stands the function does not need self as it adds no functionality.